piptools_requirements.in

```diff
@@ -0,0 +1 @@
+certifi==2018.1.18
```

Comment:
Why is this in piptools_reqs as opposed to the standard reqs file?

Reply:
When I add it to reqs file, the compilation fails.